internal/tart/vm.go

```diff
@@ -195,12 +184,29 @@ func (vm *VM) OpenSSH(ctx context.Context, config Config) (*ssh.Client, error) {
 		},
 	}
 
-	sshConn, chans, reqs, err := ssh.NewClientConn(netConn, addr, sshConfig)
-	if err != nil {
+	var sshClient *ssh.Client
+
+	if err := retry.Do(func() error {
+		dialer := net.Dialer{}
+
+		netConn, err := dialer.DialContext(ctx, "tcp", addr)
+		if err != nil {
+			return err
+		}
+
+		sshConn, chans, reqs, err := ssh.NewClientConn(netConn, addr, sshConfig)
+		if err != nil {
+			return err
+		}
+
+		sshClient = ssh.NewClient(sshConn, chans, reqs)
+		return err
```

Comment:
Can probably just return nil here?

```suggestion
		return nil
```